runtime-modules/storage/src/lib.rs

```diff
@@ -4331,17 +4283,23 @@ impl<T: Trait> Module<T> {
             .collect()
     }
 
-    fn compute_net_prize(
-        init_value: NetDeletionPrize<T>,
+    fn compute_deletion_prizes(
+        deletion_prize_request: BalanceOf<T>,
+        deletion_prize_refund: BalanceOf<T>,
         num_obj_to_create: usize,
         num_obj_to_delete: usize,
-    ) -> NetDeletionPrize<T> {
+    ) -> (BalanceOf<T>, BalanceOf<T>) {
         let amnt = Self::data_object_deletion_prize_value();
```

Comment:
```suggestion
        let object_deletion_prize = Self::data_object_deletion_prize_value();
```
I have chosen a wrong name when I implemented this line. Can you change to something with a more explicit name?

Reply:
Changed to `obj_deletion_prize`